src/sales/fraud-protection.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: Fraud Protection
+redirect_from:
+  - /sales/fraud-protection-signifyd.html
+  - /configuration/sales/fraud-protection.html
 ---
 
 Fraud protection services and filters examine submitted orders before the transaction is processed to detect fraudulent orders and protect you from the expense of chargebacks. Magento supports the following fraud protection solutions:
 
 - [PayPal Fraud Management Filter]({% link payment/paypal-fraud-management-filter.md %})
 
-- [Signifyd Guaranteed Fraud Protection]({% link sales/fraud-protection-signifyd.md %})
-
 - [Fraud Protection Solutions on Magento Marketplace][1]
 
+{:.bs-callout-info}
+To support updates for security compliance, Signifyd fraud protection is removed from Magento starting with the 2.4.x release. If you have been using the Signifyd integration in 2.3.x or a previous release, we recommend that you transition to the [Signifyd Fraud & Chargeback Protection extension](https://marketplace.magento.com/signifyd-module-connect.html){:target="_blank"} that is available on Magento Marketplace and maintain updates for the extension according to the vendor guidelines.
```

Comment:
i think if we say "starting with" it is safe to use the release version without the .x.